src/cloud/project/project-enable-mfa-enforcement.md

```diff
@@ -143,7 +179,7 @@ To authenticate and reload the SSH certificate:
 
 When MFA enforcement is enabled on a {{site.data.var.ece}} project, automated processes that require SSH access to a Cloud environment must authenticate using an API token.
 
-The API token allows the SSH connection request to bypass the user authentication flow that requires a 2FA token and SSH certificate. You generate the token from a {{site.data.var.ece}} account with Admin or Contributor access on the project.
```

Comment:
Perhaps this should have at the end -- You must generate the token after logging  into the account using 2FA credentials.

Reply:
Sure, will add that portion back.